cmd/kube-proxy/app/server.go

```diff
@@ -507,7 +510,8 @@ with the apiserver API to configure the proxy.`,
 	var err error
 	opts.config, err = opts.ApplyDefaults(opts.config)
 	if err != nil {
-		klog.Fatalf("unable to create flag defaults: %v", err)
+		klog.ErrorS(err, "Unable to create flag defaults")
+		os.Exit(1)
```

Comment:
what is the Fatalf equivalence with the logging struct change?
https://github.com/kubernetes/klog/blob/79f9f6b8291d38aaf33a0998e65a5bc254d51866/klog.go#L1511
we should have an unique way to translate these things for consistency
```
// Fatalf logs to the FATAL, ERROR, WARNING, and INFO logs,
// including a stack trace of all running goroutines, then calls os.Exit(255).
// Arguments are handled in the manner of fmt.Printf; a newline is appended if missing.
func Fatalf(format string, args ...interface{}) {
	logging.printf(fatalLog, logging.logr, logging.filter, format, args...)
}
```

Reply:
yeah, we refer to
https://github.com/kubernetes/community/blob/master/contributors/devel/sig-instrumentation/migration-to-structured-logging.md.
@yxxhero, In `user-facing change`, you should add `ACTION REQUIRED: Exit code changed from 255 to 1`

Reply:
perfect, thanks :+1: